ausias-bertran/challenge/game-of-life/script.js

```diff
@@ -0,0 +1,160 @@
+let grid = setGrid(30, 50);
+let maxRow = grid.length;
+let maxColumn = grid[0].length;
+const allCells = document.getElementsByClassName("grid");
+const cell = document.getElementsByClassName("cell");
+
+grid[2][1] = 1;
+grid[2][2] = 1;
+grid[2][3] = 1;
+
+paintCurrentGeneration();
+
+function startGame() {
+  setInterval(function () {
+    playGame();
+  }, 500);
+}
+
+function playGame() {
+  grid = gameOfLife();
+  paintCurrentGeneration();
+}
+
+function paintCurrentGeneration() {
+  for (let i = 0; i < maxRow; i++) {
+    for (let j = 0; j < maxColumn; j++) {
+      let cell = document.createElement("div");
+      cell.className = `cell`;
+      cell.style.width = `${100 / maxColumn}vw`;
+      cell.style.height = `${75 / maxRow}vh`;
+      cell.style.borderRadius = `50%`;
+      if (grid[i][j] === 1) {
+        cell.style.backgroundColor = "silver";
+      }
+      if (grid[i][j] === 0) {
+        cell.style.backgroundColor = "ghostwhite";
+      }
+      allCells[0].appendChild(cell);
+    }
+  }
+}
+
+function setGrid(rows, columns) {
+  const columnArray = [];
+  const rowArray = [];
+  for (let i = 0; i < rows; i++) {
```

Comment:
En este doble for, no indicas el length de los arrays ¿? Te funciona

Reply:
Funciona!! Como uso el valor más de una vez están declarados con scope global fuera de la función.

Reply:
Certo! ;)